cob_twist_controller/include/cob_twist_controller/constraints/constraint.h

```diff
@@ -250,6 +250,53 @@ class JointLimitAvoidanceIneq : public ConstraintBase<T_PARAMS, PRIO>
 };
 /* END JointLimitAvoidanceIneq **************************************************************************************/
 
+/* BEGIN JointSingularityAvoidance ************************************************************************************/
+/// Class providing methods that realize a JointSingularityAvoidance constraint.
+template <typename T_PARAMS, typename PRIO = uint32_t>
+class JointSingularityAvoidance : public ConstraintBase<T_PARAMS, PRIO>
```

Comment:
@ipa-bfb 
```JointSingularityAvoidance``` is a ```ConstraintBase``` so no ```DampingBase```!
To me this sounds like the JSA should be configurable as as Constraint and thus not mess up with the DampingMethods, i.e. in the cfg